src/Stryker.Core/Stryker.Core/Helpers/RoslynHelper.cs

```diff
@@ -21,31 +25,44 @@ public static TypeSyntax ReturnType(this BaseMethodDeclarationSyntax baseMethod)
                 OperatorDeclarationSyntax operatorSyntax => operatorSyntax.ReturnType,
                 _ => null
             };
-        }
 
-        public static bool NeedsReturn(this BaseMethodDeclarationSyntax baseMethod)
-        {
-            return baseMethod switch
+        /// <summary>
+        /// Checks if a method ends with return.
+        /// </summary>
+        /// <param name="baseMethod">method to analyze.</param>
+        /// <returns>true if <paramref name="baseMethod"/>is non void method, an operator or a conversion operator.</returns>
+        public static bool NeedsReturn(this BaseMethodDeclarationSyntax baseMethod) =>
+            baseMethod switch
             {
                 MethodDeclarationSyntax method => !(method.ReturnType is PredefinedTypeSyntax predefinedType &&
                                                     predefinedType.Keyword.IsKind(SyntaxKind.VoidKeyword)),
                 OperatorDeclarationSyntax _ => true,
                 ConversionOperatorDeclarationSyntax _=> true,
                 _ => false
             };
-        }
 
+        /// <summary>
+        /// Checks if a member is static
+        /// </summary>
+        /// <param name="node">Member to analyze</param>
+        /// <returns>true if it is a static method, properties, fields...</returns>
         public static bool IsStatic(this MemberDeclarationSyntax node) => node.Modifiers.Any(x => x.Kind() == SyntaxKind.StaticKeyword);
 
-        public static bool NeedsReturn(this AccessorDeclarationSyntax baseMethod)
-        {
-            return baseMethod.Keyword.Text switch
-            {
-                "get" => true,
-                _ => false
-            };
-        }
+        /// <summary>
+        /// Checks if an accessor is a getter and needs to end with a return
+        /// </summary>
+        /// <param name="baseMethod"></param>
+        /// <returns>true if this is a getter</returns>
+        public static bool NeedsReturn(this AccessorDeclarationSyntax baseMethod) =>
+            baseMethod.Keyword.Text == "get";
 
+        /// <summary>
```

Comment:
Not the most helpful doc

Reply:
Thanks. Finished documenting all helpers.